hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/CallOutcome.java

```diff
@@ -90,10 +90,10 @@ public boolean isSuccess() {
      * @param recordBuilder the record builder
      */
     public void addCallDetailsTo(@NonNull final ContractCallRecordBuilder recordBuilder) {
-        recordBuilder.contractID(recipientId);
-        // Intentionally omit result on aborted calls (i.e., when gasUsed = 0)
+        recordBuilder.contractCallResult(result);
```

Comment:
Afaict this will break fuzzy-matching for `ContractCreateSuite`...we probably need another special case when `gasUsed == 0`. 🤔